frontend/src/main/scala/scalatags/rx/nodes.scala

```diff
@@ -0,0 +1,60 @@
+package scalatags.rx
+
+import java.util.concurrent.atomic.AtomicReference
+
+import org.scalajs.dom
+import org.scalajs.dom.Element
+import org.scalajs.dom.ext._
+import org.scalajs.dom.raw.Comment
+import rx._
+
+import scala.collection.immutable
+import scala.language.implicitConversions
+import scalatags.JsDom.all._
+import scalatags.jsdom
+import scalatags.rx.ext._
+
+trait RxNodeInstances {
+
+  implicit class rxStringFrag(v: Rx[String])(implicit val ctx: Ctx.Owner) extends jsdom.Frag {
+    def render: dom.Text = {
+      val node = dom.document.createTextNode(v.now)
+      v foreach { s => node.replaceData(0, node.length, s) } attachTo node
+      node
+    }
+  }
+
+  implicit class bindRxElement[T <: dom.Element](e: Rx[T])(implicit val ctx: Ctx.Owner) extends Modifier {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Class name does not match the regular expression '^[A-Z][A-Za-z0-9]*$'](https://www.codacy.com/app/lustefaniak/akka-viz/file/2215957607/issues/source?bid=3121282&fileBranchId=3172228#l27)